civis/ml/_model.py

```diff
@@ -527,6 +539,15 @@ class ModelPipeline:
     notifications : dict
         See :func:`~civis.resources._resources.Scripts.post_custom` for
         further documentation about email and URL notification.
+    dependencies : array, optional
+        List of packages to install from PyPI or Github. If a private Github
+        package is specificed, please include a ``github_token_name``
+        argument as well (see below).
+    github_token_name : str, optional
+        Name of Github API token stored in platform as the password field in a
+        custom platform credential. Used only when installing private Github
+        repositories. Note, these API tokens can be generated from this URL:
+        https://github.com/settings/tokens.
```

Comment:
This could potentially be a token from another service as well, couldn't it? E.g. https://blog.bitbucket.org/2016/06/06/app-passwords-bitbucket-cloud/ ?